### PR TITLE
Docs homepage, nav change, case study style

### PR DESCRIPTION
**Pull request by jathayde**
This PR contains navigation changes as discussed with Amanda, including a new Docs homepage and linking out to the new Merch store. Navigation now highlights based on the page you are on.

There is a new docs homepage linking out to various locations, and the case studies area to link to future Jekyll pages. The Doximity is held here with a "COMING SOON" banner.

We may want to 301 redirect old `/blog` urls to the new /news.

### CODE REVIEW

**Comment by AmandaPerino** [comment]
Adding screenshots: 
Screenshot 2024-09-30 at 10 32 12 PM
Screenshot 2024-09-30 at 10 31 56 PM

**Comment by AmandaPerino** [comment]
@jathayde Are we able to already add the blog redirects to this PR? We have a lot of links in the wild that I wouldn't want to interfere with.

**Comment by dhh** [comment]
Where is the actual doximity case study living?

Also, wonder if the forum should live under Community instead of under Docs? But don't have that strong opinions about it either way.

In terms of design, I think the desktop one is fine, but the mobile one looks pretty inefficient with it's space. This is basically just a page of links. To only be able to fit one link per visible scroll on mobile doesn't seem quite right. I'd just kill the navigation logos at that size.

**Comment by jathayde** [comment]
> @jathayde Are we able to already add the blog redirects to this PR? We have a lot of links in the wild that I wouldn't want to interfere with.

I'll work on that and add here.

**Comment by dhh** [comment]
Okay. I'm not sure it makes sense to actually screw with the links here, btw. Even with a 301. We can just call the link News instead of blog, but retain blog in the URL. Then we don't risk screwing with search engines and old links and whatever. And it doesn't matter anyway.

But I would also not launch this until there's some actual content to show for it. Moving the links around should be in service of some actual new stuff. Does not make sense to me without that.

**Comment by zzak** [comment]
> Will take a stab at a few options to reduce the vertical space taken up by the graphics.

Maybe you can shrink the logo and put it on the top left, and float that header in the top middle and shrink it too. It looks like the logo is the same size on mobile and desktop, I think more detail on the smaller screen size is preferable.

Is the merch site also a work in progress? How does that line up to releasing this?

Thanks for your hard work!

**Comment by AmandaPerino** [comment]
> I think the team preferred "News" over blog since it is more appropriate title, but does having the label of a link not line up with the URL or content matter for SEO?

I could be wrong, but don't think that matters so much for SEO.

> Is the merch site also a work in progress? How does that line up to releasing this?

Merch store is ready to go. The Doximity case study and the tutorial will take a bit more work, so we have a bit of time to get everything ready to go live together.

File: _pages/docs.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+      <div class="text__content common-content">
+
+        <h2 class="thin">Get Started</h2>
+        <ul class="learning-cards">
+          <li>
+            <a href="https://guides.rubyonrails.org/getting_started.html#creating-a-new-rails-project-installing-rails" title="Install Ruby on Rails">
+              <img src="/assets/images/docs-banner-installation.svg" alt="Install Ruby on Rails">
+            </a>
+            <h4>Installation</h4>
+            <p>Get Ruby on Rails installed and running properly on your system, whether it be Windows, MacOS, or Linux.</p>
```
Comment (robin850) [inline _pages/docs.html:35]:
```suggestion
            <p>Get Ruby on Rails installed and running properly on your system, whether it be Windows, macOS, or Linux.</p>
```

File: _pages/docs.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            </a>
+            <h4>Tutorial</h4>
+            <p>From zero to hero, learn the basics of the Ruby on Rails framework and how it enables you to be a better developer.</p>
+          </li>
+          <li>
+            <a href="https://guides.rubyonrails.org/" title="The Ruby on Rails Guides">
+              <img src="/assets/images/docs-banner-guides.svg" alt="The Ruby on Rails Guides">
+            </a>
+            <h4>Guides</h4>
+            <p>The Rails Guides covers all facets of the Rails ecosystem, including the primary gems (libraries) and functionality that is available out of the box.</p>
```
Comment (robin850) [inline _pages/docs.html:49]:
```suggestion
            <p>The Rails Guides cover all facets of the Rails ecosystem, including the primary gems (libraries) and functionalities that are available out of the box.</p>
```

File: _pages/docs.html [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        <ul class="learning-cards">
+          <li>
+            <a href="https://api.rubyonrails.org/" title="The API Documentation">
+              <img src="/assets/images/docs-banner-api.svg" alt="API Documentation">
+            </a>
+            <h4>API</h4>
+            <p>The full framework reference, from A to Z for every piece of Rails.</p>
+          </li>
+          <li>
+            <a href=https://discuss.rubyonrails.org/" title="The Rails Forum">
```
Comment (robin850) [inline _pages/docs.html:63]:
```suggestion
            <a href="https://discuss.rubyonrails.org/" title="The Rails Forum">
```

File: _pages/docs.html [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+            </a>
+            <h4>Forum</h4>
+            <p>Join the conversation and ask for help on the Rails Forum.</p>
+          </li>
+          <li>
+            <a href="https://github.com/rails/rails" title="Contribute to Rails">
+              <img src="/assets/images/docs-banner-contribute.svg" alt="Contribute to Rails">
+            </a>
+            <h4>Contribute</h4>
+            <p>Rails is open-source, and that means anyone can contribute. Visit Github to see the code & learn more.</p>
```
Comment (robin850) [inline _pages/docs.html:74]:
```suggestion
            <p>Rails is open-source, and that means anyone can contribute. Visit GitHub to see the code & learn more.</p>
```

File: _pages/docs.html [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+        <p class="how-leadin">
+          See how some of the most successful online products used
+          Rails to launch, scale, and thrive in a competitive world.
+          Read these case studies to learn more:
+        </p>
+
+        <ul class="case-studies">
+          <li class="case-study">
+            <div class="corner-ribbon">Coming Soon!</div>
+              <img src="/assets/images/logo-doximity.svg" alt="Doximity"s>
```
Comment (robin850) [inline _pages/docs.html:96]:
```suggestion
              <img src="/assets/images/logo-doximity.svg" alt="Doximity">
```

**Comment by jathayde** [comment]
> Also, I'd prefer if we kept the style for the headline in line with the rest of the site. Don't think this needs to be broken into three lines for stylistic reasons.

In original, unaltered h1 style we break the line:

Screenshot 2024-10-03 at 12 22 05 PM

I'd like to at least control that to look like this:

Screenshot 2024-10-01 at 5 56 10 PM

on mobile, we'd `display: none` on the `<br>` and let it wrap naturally:

Screenshot 2024-10-03 at 12 22 44 PM

**Comment by jathayde** [comment]
> > Will take a stab at a few options to reduce the vertical space taken up by the graphics.
> 
> Maybe you can shrink the logo and put it on the top left, and float that header in the top middle and shrink it too. It looks like the logo is the same size on mobile and desktop, I think more detail on the smaller screen size is preferable.

This is how it is in production today. I can take a look at this and see what we can do, but reducing that top 150px to maybe 75 would help focus more on the content. 

> Also on mobile, maybe the section banners could just be the outline of the emoji next to the relevant section header. It's very red right now.

Agreed. Working on two different options to reduce both the red and the lost space/inefficency per your and @dhh's notes 

Thank you for the feedback!

**Comment by jathayde** [comment]
Here's an option for the homepage on mobile, which dramatically reduces the vertical space but retains the icons without having additional graphics. 

Screenshot 2024-10-04 at 4 06 58 PM

**Comment by dhh** [comment]
That's certainly a lot better. But I think it also calls into question what the value of those icons really is. But still, big improvement.

**Comment by dhh** [comment]
Something else is that we need to retain a consistent capitalization scheme. "How Did They Do It" stands out from the entire rest of the site which uses "You're in good company" etc.

**Comment by jathayde** [comment]
Revised using visual elements from other parts of the existing site, removing icons entirely, and some copy help from @AmandaPerino to balance the lines in the boxes better and reduce repitition.

Screenshot 2024-10-15 at 10 03 08 AM

**Comment by AmandaPerino** [comment]
> I do have a general concern that a significant segment of repeat visitors are going to be looking exclusively for the API docs or maybe guides, and beyond the extra click, it feels like they're floating in the middle of a large space. I'm not sure what I'd suggest to do differently, though. 😕

The only thing I can think to do here to address that is to create a dropdown from that navbar tab to make it a more direct/fewer clicks experience, but not sure if that's ideal either.

However, my guess is that most devs would just start typing "api" in the URL and the API docs would load based on previous visits, bypassing the homepage and docs landing page altogether.

**Comment by jathayde** [comment]
I feel from a flow perspective, most people aren't coming to this page as power users. They have their bookmarks, muscle memory, or other means of consuming the docs (e.g. Dash on macOS is my go to for API reference). We can provide a drop down, but we don't do that on other items, so I would ask if it is truly needed.

**Comment by zzak** [comment]
I'm not sure I like the card style on mobile, it eats up so much space, but I also didn't love the gradient style on the icon version either. :thinking: 

> However, my guess is that most devs would just start typing "api" in the URL and the API docs would load based on previous visits, bypassing the homepage and docs landing page altogether.

This is me, but I also think the dropdown is fine ala [vuejs](https://vuejs.org/guide/introduction.html) but they don't have a docs landing page. Also looking at [swift](https://www.swift.org/getting-started/), they have a separate page for "get started" and a docs landing page which points to api reference, etc. Kind of making the choice down to "are you new here?" or "do you know what you're looking for?"

**Comment by jathayde** [comment]
Case study styling:
Screenshot 2024-12-06 at 11 10 47 AM
Screenshot 2024-12-06 at 11 10 51 AM

**Review by AmandaPerino — APPROVED** [review]
Updated the case study copy based on recent changes.

File: _pages/docs.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (jathayde) [reply _pages/docs.html]: image
 I'm seeing this on the contribute box. What should the description string be?

[the remark this answers is not in the document]
Comment (AmandaPerino) [reply _pages/docs.html]: Sorry, when I pushed the copy edits live just now, I apparently had a few comments that I hadn't submitted yet from October 1st. So this is likely from an outdated version.

It should be:

**Get involved**
Contribute to Rails
Report issues, review code, or submit updates to the open source Rails project.

File: _case_studies/doximity.html [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+    </div>
+  </div>
+</div>
+
+<div class="docs casestudy cta text common-padding--bottom">
+  <div class="container">
+    <div class="text__body">
+      <div class="text__content common-content">
+        <p>
+          Experience the power Rails can bring to your engineering efforts firsthand. Download Rails and see how you can take your idea from Startup to IPO.
```
Comment (AmandaPerino) [inline _case_studies/doximity.html:227]:
```suggestion
          Try it yourself. Download Rails and see how you can take your idea from Startup to IPO.
```

Comment (AmandaPerino) [reply _case_studies/doximity.html:227]: Updating the CTA at the bottom of the page.

**Review by AmandaPerino — APPROVED** [review]

**Review by rafaelfranca — APPROVED** [review]
@AmandaPerino feel free to merge

**Pull request merged**